packages/front-end/components/Metrics/DateGraph.tsx

```diff
@@ -178,11 +129,12 @@ const DateGraph: FC<{
         ) : (
           <>
             <div className={styles.val}>
-              &mu;: {formatConversionRate(type, d.v as number)}
+              {method === "sum" ? `Σ` : `μ`}:{" "}
```

Comment:
uhh, is putting these characters directly into the `.tsx` the right approach? This feels risky to me but I have no idea what the preferred approach is these days.